Services/Dashboard/Administration/classes/class.ilObjDashboardSettingsGUI.php

```diff
@@ -204,153 +317,107 @@ public function setSettingsSubTabs(string $a_active): void
-        );
-
-        return $form;
-    }
-
-
-    protected function saveViewCoursesGroups(): void
-    {
-        $this->saveViewSettings(
-            $this->viewSettings->getMembershipsView(),
-            "editViewCoursesGroups"
+            $title
         );
     }
 
-    protected function editViewFavourites(): void
+    protected function savePresentation(): void
     {
-        $main_tpl = $this->tpl;
-        $tabs = $this->tabs_gui;
-        $ui_renderer = $this->ui_renderer;
-
-        $tabs->activateTab("settings");
-        $this->setSettingsSubTabs("view_favourites");
+        $request = $this->request;
+        $lng = $this->lng;
+        $ctrl = $this->ctrl;
 
-        $view = $this->viewSettings->getSelectedItemsView();
+        $form = $this->getViewForm(self::VIEW_MODE_PRESENTATION);
+        $form = $form->withRequest($request);
+        $form_data = $form->getData();
 
-        $form = $this->getViewSettingsForm($view);
+        if (!$this->canWrite()) {
+            $this->tpl->setOnScreenMessage('failure', $this->lng->txt('no_permission'), true);
+            $this->editPresentation();
+        }
 
-        $main_tpl->setContent($ui_renderer->render($form));
-    }
 
-    protected function saveViewFavourites(): void
-    {
-        $this->saveViewSettings(
-            $this->viewSettings->getSelectedItemsView(),
-            "editViewFavourites"
-        );
+        foreach ($form_data as $view => $view_data) {
+            $this->viewSettings->storeViewPresentation(
+                $view,
+                $view_data['default_pres'],
+                $view_data['avail_pres'] ?: []
+            );
+        }
+        $this->tpl->setOnScreenMessage('success', $lng->txt("msg_obj_modified"), true);
+        $this->editPresentation();
     }
 
-    protected function saveViewSettings(int $view, string $redirect_cmd): void
+    public function saveSorting(): void
     {
-        $request = $this->request;
-        $lng = $this->lng;
-        $ctrl = $this->ctrl;
-
         if (!$this->canWrite()) {
-            ilUtil::sendFailure($this->lng->txt('no_permission'), true);
-            $ctrl->redirect($this, $redirect_cmd);
+            $this->tpl->setOnScreenMessage('failure', $this->lng->txt('no_permission'), true);
+            $this->editSorting();
         }
 
-        $form = $this->getViewSettingsForm($view);
-        $form = $form->withRequest($request);
+        $form = $this->getViewForm(self::VIEW_MODE_SORTING);
+        $form = $form->withRequest($this->request);
         $form_data = $form->getData();
-        $this->viewSettings->storeViewSorting(
-            $view,
-            $form_data['sortation']['default_sort'],
-            $form_data['sortation']['avail_sort'] ?: []
-        );
-        $this->viewSettings->storeViewPresentation(
-            $view,
-            $form_data['presentation']['default_pres'],
-            $form_data['presentation']['avail_pres'] ?: []
```

Comment:
Do you want to proof `== false` or `=== null` here?
If second, use `??` since its more strict.